api/utils.py

```diff
@@ -3,53 +3,62 @@
 from urllib.request import Request, urlopen
 
 import orjson
+import i18n
 
+i18n.set("file-format", "yaml")
+i18n.set("filename_format", "{locale}.{format}")
+i18n.set("enable_memoization", True)
+i18n.load_path.append("./api/locale")
+_ = i18n.t
```

Comment:
The `_` convention is what's used by the gettext module and a lot of older python files that relied on that format for localising strings (prior to using the python-i18n module, I implemented localisation with gettext, but that required generating POT files).
But, I guess for the library it would make more sense to use `i18n.t`.